### PR TITLE
Remove dependency on ~/OpenROAD-flow-scripts

### PR DESCRIPTION
**Pull request by eszpotanski**
This PR removes dependency on ~/OpenROAD-flow-scripts.

Instead, `ORFS` variable is used to store location of OpenROAD-flow-scripts.

### CODE REVIEW

File: orfs [hunk 1]
```diff
@@ -2,7 +2,11 @@
 set -ex
 WORKSPACE="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-export ORFS=~/OpenROAD-flow-scripts
```
Comment (oharboe) [inline orfs:5]: Use the `. env.sh` variable convention, FLOW_HOME, no need to introduce a the ORFS variable:

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/470955ddac63b2b571e6a83e7da8734cd7822397/env.sh#L25

**Review by oharboe — CHANGES_REQUESTED** [review]
Use ORFS's `. env.sh` https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/470955ddac63b2b571e6a83e7da8734cd7822397/env.sh#L25 FLOW_HOME variable, don't introduce a new "ORFS" env var.

File: orfs (hunk 1 above)
Comment (eszpotanski) [reply orfs:5]: `ORFS` removed, now only `FLOW_HOME` is used

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This repository contains [Bazel](https://bazel.build/) rules for wrapping Physic
 ## Requirements
 
 * [Bazelisk](https://bazel.build/install/bazelisk) or [Bazel](https://bazel.build/install) - if using `bazel`, please refer to `.bazelversion` file for the recommended version of the tool.
-* [OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts) - **Required only for running `Make` targets** - must reside under `~/OpenROAD-flow-scripts`. `bazel-orfs` intentionally does not treat OpenROAD-flow-scripts as a installable versioned tool, but prefers to rely on `~/OpenROAD-flow-scripts` such that it is easy to hack ORFS and OpenROAD.
+* [OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts) - **Required only for running `Make` targets** - to use it, `env.sh` file from OpenROAD-flow-scripts has to be sources or `FLOW_HOME` environmental variable has to be set manually to `OpenROAD-flow-scripts/flow` location. `bazel-orfs` intentionally does not treat OpenROAD-flow-scripts as a installable versioned tool, but prefers to rely on local installation such that it is easy to hack ORFS and OpenROAD.
```
Comment (oharboe) [inline README.md:11]: spelling: sources -> sourced

**Review by oharboe — CHANGES_REQUESTED** [review]
spelling nit

File: README.md (hunk 2 above)
Comment (eszpotanski) [reply README.md]: fixed

**Pull request merged**